tests/Unit/Utilities/Test_Functional.cpp

```diff
@@ -136,6 +141,8 @@ MAKE_UNARY_TEST(Tanh, tanh);
 MAKE_BINARY_TEST(Atan2, atan2);
 MAKE_BINARY_TEST(Hypot, hypot);
 MAKE_BINARY_TEST(Pow, pow);
+MAKE_BINARY_TEST(Max, max);
+MAKE_BINARY_TEST(Min, min);
```

Comment:
Looks like we are trying to keep these alphabetical

Reply:
Done :)